nuget/bootstrap.nuspec

```diff
@@ -16,9 +16,9 @@
     <copyright>Copyright 2017</copyright>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <dependencies>
-      <dependency id="jQuery" version="3.0.0.1" />
+      <dependency id="jQuery" version="[1.9.1,3)" />
```

Comment:
in [nuget packages](https://docs.nuget.org/ndocs/create-packages/dependency-versions#version-ranges) yes